src/menu/menu.c

```diff
@@ -1,57 +1,103 @@
 #include <stdio.h>
 #include <string.h>
+#include <sys/stat.h>
+#include <unistd.h>
 
-#include <fatfs/ff.h>
 #include <libdragon.h>
 
 #include "flashcart/flashcart.h"
+#include "libs/toml/toml.h"
 
 #include "menu.h"
 
 
-static char *config_path = "n64/config.txt";
-static char game_path[256];
-static char save_path[256];
+#define SC64_CONFIG_FILEPATH "sd://config.sc64.toml.txt"
```

Comment:
Why `.toml.txt`? The standard expects the file extension to be `.toml`, and I think people used to this format would also expect that.

Reply:
for the moment (at least whilst in development) it is easier on windows to edit the file.
I would expect the extension and name/location to be changed before going gold.